### PR TITLE
SW-4483 Add mandatory notes to accession quantity edits

### PR DESCRIPTION
**Pull request by karthikbtf**
- added notes to the edit modal
- pass notes along with the update API request

Terraware App 2023-12-05 10-38-31

Terraware App 2023-12-05 10-38-47

### CODE REVIEW

**Review by charlie-tf — APPROVED** [review]

**Comment by karthikbtf** [comment]
@charlie-tf I made a few changes since your last review.. take a look when you get a chance?

**Review by charlie-tf — APPROVED** [review]
Minor question, but otherwise looks great!

File: src/components/accession2/edit/QuantityModal.tsx [hunk 1]
```diff
@@ -80,6 +80,13 @@ export default function QuantityModal(props: QuantityModalProps): JSX.Element {
   const [remainingQuantityNotesError, setRemainingQuantityNotesError] = useState<boolean>(false);
   const classes = useStyles();
 
+  const quantityChanged = useMemo(() => {
+    return (
+      record.remainingQuantity?.quantity?.toString() !== accession.remainingQuantity?.quantity?.toString() ||
```
[the remark this answers is not in the document]
Comment (karthikbtf) [reply src/components/accession2/edit/QuantityModal.tsx:85]: we have a mixed representation of number and string depending on edit vs read-only

[the remark this answers is not in the document]
Comment (charlie-tf) [reply src/components/accession2/edit/QuantityModal.tsx:85]: Gotcha, makes sense

**Pull request merged**